test-rpm-build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# $1: Path to a `xcp-ng-rpms` repo to test its build
+# TARGET_XCP_NG_VERSION: env var used to define the XCP-ng version to used (eg: 8.2)
+# CONTAINER_NAME: env var used to define the name of the container created by the env builder
+
+set -eux
+
+BUILD_ENV_PATH=/tmp/build-env
+mkdir -p "$BUILD_ENV_PATH"
+
+git clone git@github.com:xcp-ng/xcp-ng-build-env.git "$BUILD_ENV_PATH"
+
+TARGET_XCP_NG_VERSION=${TARGET_XCP_NG_VERSION:-8.2}
```

Comment:
Just a question: Do you think we must use a hardcoded `8.2` version here?

Reply:
It can be overwriten by using `TARGET_XCP_NG_VERSION=x.y ./test-rpm-build.sh ...`.